src/CodeMirrorBlocks.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import ToggleEditor from './ui/ToggleEditor';
-import wescheme from '../src/languages/wescheme';
+import pyret from '../src/languages/pyret';
```

Comment:
We should include _both_ languages here -- no reason for the constructor to only know about one of them!

Reply:
This is for the purpose of the default constructor. If we imported both, one of them would be unused.

Reply:
But I agree that we should remove this or document this better when other languages start using CMB.